en/asset/auto-atlas.md

```diff
@@ -30,9 +30,18 @@ After selecting an **Auto Atlas Resource** in the __Assets Panel__, the **Proper
 After the configuration is complete, you can click the **Preview** button to preview the packaged results. The related results generated according to the current automatic atlas configuration will be displayed in the area below the **Properties Inspector**. Please note that after each configuration, you can re-click **Preview** to update the preview image. (Generating a preview is not required).
 
 The results are divided into:
-   - __Packed Textures:__ Display the packaged atlas pictures and picture-related information. If there are multiple pictures to be generated, they will be listed below in the **Property Inspector**.
-   - __Unpacked Textures:__ Display the broken image assets that cannot be packed into the atlas. The cause may be that the size of these broken image assets is larger than the size of the atlas assets. At this time, the configuration or fragmentation of the following atlas may need to be adjusted. The size of the figure is increased.
+
+- __Packed Textures:__ Display the packaged atlas pictures and picture-related information. If there are multiple pictures to be generated, they will be listed below in the **Property Inspector**.
+- __Unpacked Textures:__ Display the broken image assets that cannot be packed into the atlas. The cause may be that the size of these broken image assets is larger than the size of the atlas assets. At this time, the configuration or fragmentation of the following atlas may need to be adjusted. The size of the figure is increased.
 
 ## Generating an Atlas
 
-When inside the editor or previewing the project __Cocos Creator 3D__ is directly using the split **SpriteFrame** assets, only after you build the project with the option **AutoAtlas** enabled, the Atlas asset will be generated and be used instead of all split assets.
\ No newline at end of file
+When inside the editor or previewing the project __Cocos Creator 3D__ is directly using the split **SpriteFrame** assets, only after you build the project with the option **AutoAtlas** enabled, the Atlas asset will be generated and be used instead of all split assets.
+
+In general, after Atlas asset is generated, the Texture2D assets and Image assets related by the original split assets will be deleted in the package. The following two special cases will have special process:
+
+1. When Atlas asset is in a `resources` directory, the Texture2D assets and Image assets related by the original SpriteFrame assets in the AutoAtlas asset's directory will also be generated.
+
+2. When any Texture2D asset depended by SpriteFrame assets in the Atlas asset's folder is directly used by other asset, the dependent **Texture2D** asset and **Image** asset will be generated also.
```

Comment:
When any Texture2D asset depended by SpriteFrame assets in the Atlas asset's folder is directly used by other asset, the dependent **Texture2D** asset and **Image** asset will be generated also.
-->
When any **Texture2D **asset depended on by **SpriteFrame** assets in the **Atlas** asset's folder is directly used by another asset, the dependent **Texture2D** asset and **Image** asset will also be generated.